frontend/src/bbkit/BBSelect.vue

```diff
@@ -113,6 +120,10 @@ export default defineComponent({
       default: false,
       type: Boolean,
     },
+    showPrefixItem: {
```

Comment:
why we need a separate showPrefixItem instead of checking whether placeholder is empty?

Reply:
I consider that not every dropdown needs a prefix item. Sometimes we do not show a prefix item to keep it simple.